From: Simon McVittie <smcv@collabora.com>
Date: Mon, 28 Jun 2021 14:29:12 +0100
Subject: build: Fix installation of zsh completions in user-specified
 directory

This copy/paste from bash could end up installing into "$(DESTDIR)yes",
which is not as intended.

Forwarded: https://github.com/containers/bubblewrap/pull/440
Signed-off-by: Simon McVittie <smcv@collabora.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 33b75cf..6fba84f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -62,7 +62,7 @@ AC_ARG_WITH([zsh-completion-dir],
 if test "x$with_zsh_completion_dir" = "xyes"; then
         [ZSH_COMPLETION_DIR="$datadir/zsh/site-functions"]
 else
-    ZSH_COMPLETION_DIR="$with_bash_completion_dir"
+    ZSH_COMPLETION_DIR="$with_zsh_completion_dir"
 fi
 
 
